.github/workflows/CI.yml

```diff
@@ -34,40 +34,129 @@ jobs:
     - run: bun run test
 
   build:
-    name: Build + Smoke Test
+    name: Build
     runs-on: ubuntu-latest
 
     needs: lint
 
+    permissions:
+      contents: read
+      packages: write
+
+    outputs:
+      image-tag: ${{ steps.tag.outputs.first }}
+      image-digest: ${{ steps.build.outputs.digest }}
+
     steps:
     - uses: actions/checkout@v4
       with:
         show-progress: false
+
     - uses: docker/setup-qemu-action@v3
     - uses: docker/setup-buildx-action@v3
+    - uses: docker/login-action@v3
+      with:
+        registry: "ghcr.io"
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        images: |
+          ghcr.io/nulldev/csz-bot
+        tags: |
+          type=raw,value=latest,enable=${{ github.ref == 'refs/heads/master' }}
+          type=ref,event=branch
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
```

Comment:
Muss man die überhaupt setzen oder überschreibt das setzen des Tags-Felds die defaults?

Reply:
Gibt keinen default, wenn du das nicht setzt, gibt's auch kein tag als output.

Reply:
Scheinbar wird der Default überschrieben, wenn man es setzt:
![image](https://github.com/user-attachments/assets/6ba7f46c-e22f-4ad7-8dbc-a1bef964bca7)

Reply:
Ah okay, der default ist hier: https://github.com/docker/metadata-action/blob/60a0d343a0d8a18aedee9d34e62251f752153bdb/src/tag.ts#L56-L64, nicht in der `action.yml`